src/Engine/ProtoCore/Lang/BuiltInMethods.cs

```diff
@@ -12,119 +12,39 @@ public class BuiltInMethods
         public enum MethodID
         {
```

Comment:
kind of worried about this as well, in this case atleast the enum is still present... I'm not sure of the consequence of this.

Reply:
I guess I would prefer we just obsolete each member and remove it at 3.0.

Reply:
Ok, will make the change.